src/TodoApp/TodoApp.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import { TitleComponent } from './components/Title/TitleComponent';
+import { AddTodoComponent } from './components/AddTodo/AddTodoComponent';
+import { TodoListComponent } from './components/TodoList/TodoListComponent';
+import { getTodos, saveTodos } from './helpers/store';
+
+const s4 = () => {
```

Comment:
```suggestion
const getId = () => {
```

There could be a better name for this function